lib/ui/screens/login_screen/login_screen.dart

```diff
@@ -246,6 +249,24 @@ class _LoginScreenState extends ConsumerState<_LoginScreen> {
                       ),
                     ),
                   ),
+                  const SizedBox(height: 8),
+                  Visibility(
```

Comment:
Ideally, this could be animated. Not necessarily though.

Reply:
i'm largely unfamiliar with animated uis in general so i'll leave this up to you if you want to animate it